ComfySigns/ComfySigns.cs

```diff
@@ -80,6 +81,11 @@ public class ComfySigns : BaseUnityPlugin {
 
     public static readonly EventHandler OnSignConfigChanged = (_, _) => {
       TMP_FontAsset font = UIFonts.GetFontAsset(SignDefaultTextFont.Value);
+
+      if (UseFallbackFonts.Value && font != null) {
```

Comment:
```
if (UseFallbackFOnts.Value && !font) {
  
}
```

Makes use of the Unity overloaded boolean null comparison check since UnityObjects be different.